sources/packages/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -595,32 +605,57 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
   /**
    * Get the offering to request change.
    * Offering in Approved status alone can be requested for change.
-   * @param locationId
-   * @param programId
-   * @param offeringId
+   * @param offeringId offering requested for change.
+   * @param offeringStatus status of the approval.
+   * @param programId program of the offering.
+   * @param locationId location of the offering.
    * @returns offering.
    */
-  async getApprovedOfferingToRequestChange(
-    locationId: number,
-    programId: number,
+  async getOfferingSummaryToRequestChange(
```

Comment:
`Summary` does not make sense here IMO. I believe that would be better just `getOfferingToRequestChange`.